cpp/src/streaming/core/channel.cpp

```diff
@@ -8,13 +8,15 @@
 namespace rapidsmpf::streaming {
 
 coro::task<bool> Channel::send(Message msg) {
+    RAPIDSMPF_EXPECTS(!msg.empty(), "message cannot be empty");
```

Comment:
Add to doc

Reply:
Why do you want this?

Reply:
@wence- If we cant differentiate an empty message from a channel shutdown when calling `receive`, IMO we should not allow sending empty messages